test/jdk/javax/swing/JInternalFrame/6725409/bug6725409.java

```diff
@@ -21,63 +21,53 @@
  * questions.
  */
 
-/* @test
+/*
+ * @test
  * @key headful
  * @bug 6725409
  * @requires (os.family == "windows")
  * @summary Checks that JInternalFrame's system menu
  *          can be localized during run-time
- * @author Mikhail Lapshin
- * @library /lib/client/
  * @modules java.desktop/com.sun.java.swing.plaf.windows
- * @build ExtendedRobot
  * @run main bug6725409
  */
 
-import javax.swing.*;
-import java.awt.*;
+import java.awt.Robot;
+import javax.swing.JDesktopPane;
+import javax.swing.JFrame;
+import javax.swing.JInternalFrame;
+import javax.swing.JMenuItem;
+import javax.swing.JPopupMenu;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
 
 public class bug6725409 {
     private JFrame frame;
     private JInternalFrame iFrame;
-    private TestTitlePane testTitlePane;
+    private static TestTitlePane testTitlePane;
     private boolean passed;
-    private static ExtendedRobot robot = createRobot();
+    private static final Robot robot = createRobot();
```

Comment:
Suggestion but not a must change: Since createRobot() is just creating an instance of Robot, the extra method can be removed and robot can be initialized in main method.